source/usr/lib/mcvirt/cluster/remote.py

```diff
@@ -367,7 +367,14 @@ def __connect(self):
             except AuthenticationException:
                 raise NodeAuthenticationException('Could not authenticate to node: %s' % self.name)
             except Exception:
-                raise CouldNotConnectToNodeException('Could not connect to node: %s' % self.name)
+                from mcvirt.auth import Auth
+
+                if (Auth().checkPermission(Auth.PERMISSIONS.CAN_IGNORE_CLUSTER)):
```

Comment:
If my understanding of this is correct, perhaps it would be a good idea display this option in the help text, but not suggest when it could be used since one may not know the implications of the flag.